clang/include/clang/AST/Decl.h

```diff
@@ -3206,6 +3206,10 @@ class FieldDecl : public DeclaratorDecl, public Mergeable<FieldDecl> {
   /// Set the C++11 in-class initializer for this member.
   void setInClassInitializer(Expr *NewInit);
 
+  /// Find the FieldDecl specified in a FAM's "counted_by" attribute. Returns
+  /// \p nullptr if either the attribute or the field doesn't exist.
+  const FieldDecl *FindCountedByField() const;
```

Comment:
nit: all of the other non-static member functions in this class seem to be camelCase, so this one should be as well (so we can have at least a semblance of consistency here ;Þ)